src/openxr/OpenXRApi.h

```diff
@@ -275,6 +250,8 @@ class OpenXRApi {
 	OpenXRApi();
 	~OpenXRApi();
 
+	void register_extension_wrapper(XRExtensionWrapper* wrapper);
+
```

Comment:
Where is this called to actually register our two wrappers? I'm probably overlooking the code but I don't see it...

Reply:
It's invoked in `ARVRInterface#godot_arvr_initialize()` for now. This can be moved around if the initialize call sequence for Godot 4.x will be significantly different.

Reply:
Ok cool, I guess that wasn't checked in yet or I missed it :)